packages/squiggle-lang/src/rescript/FR/FR_Example.res

```diff
@@ -0,0 +1,47 @@
+open FunctionRegistry_Core
```

Comment:
It seems weird to have an "example" in the main codebase. Could work as a test; we frequently use tests as examples.

Reply:
renamed to "template", which is more of the use case.  But can delete.

Reply:
I'd either move to docs or delete.

Reply:
Ok, deleted